projects/ngx-prx-styleguide/src/lib/tz-datepicker/tz-datepicker.component.css

```diff
@@ -7,12 +7,16 @@ input {
   outline: 5px auto #f09b4c;
 }
 
-input.ng-invalid,
```

Comment:
This was giving us an invalid outline on an empty time field because it is required. We don't show fields as invalid until after the user has edited them.